Orm/Xtensive.Orm/Orm/Linq/Expressions/Visitors/IncludeFilterMappingGatherer.cs

```diff
@@ -34,7 +34,7 @@ public MappingEntry(LambdaExpression calculatedColumn)
       }
     }
 
-    private readonly ParameterExpression calculatedColumnParameter;
+    private static readonly ParameterExpression calculatedColumnParameter = Expression.Parameter(WellKnownOrmTypes.Tuple, "filteredRow");
```

Comment:
Naming policy violation.

Reply:
what is naming policy here?
I see many lowercase private static names in the project

Reply:
fixed